Mathlib/GroupTheory/GroupAction/Basic.lean

```diff
@@ -378,6 +378,25 @@ lemma orbitRel_r_apply {a b : α} : (orbitRel G _).r a b ↔ a ∈ orbit G b :=
 lemma orbitRel_subgroup_le (H : Subgroup G) : orbitRel H α ≤ orbitRel G α :=
   Setoid.le_def.2 mem_orbit_of_mem_orbit_subgroup
 
+@[to_additive]
+lemma orbitRel_subgroupOf (H K : Subgroup G) :
+    orbitRel (H.subgroupOf K) α = orbitRel (H ⊓ K : Subgroup G) α := by
```

Comment:
Would this be useful as a `@[simp]` lemma? No need to change if uncertain.

Reply:
I don't see either side as clearly being simpler.